python/tests/test_integration.py

```diff
@@ -80,3 +80,10 @@ def test_duckdb_nan(table_test_null_nan_rewritten: Table) -> None:
     result = con.query("SELECT idx, col_numeric FROM table_test_null_nan WHERE isnan(col_numeric)").fetchone()
     assert result[0] == 1
     assert math.isnan(result[1])
+
+
+@pytest.mark.integration
+def test_ray_dataset_nan(table_test_null_nan_rewritten: Table) -> None:
```

Comment:
can we add some more tests?

Reply:
Sure! I created a new table containing all the types in the test environment and used it to test the conversion from iceberg to ray dataset